### PR TITLE
Implement 5.15 from NEN-EN 1992-1-1+C2:2011

### LINKED ISSUE #167: [✨ Feature request]: Add formula 5.15 from NEN_EN_1992_1_1_C2_2011
### Description

Add formula 5.15 from NEN_EN_1992_1_1_C2_2011

Image
afbeelding

### Acceptance Criteria
- [ ] Add the formula to the code
- [ ] Add latex expression to the formula
- [ ] Add unittests
- [ ] Add docstrings
- [ ] Update table in docs with :heavy_check_mark:

### Dependencies

_not applicable_

### Confirmation

- [X] I have checked that this feature does not already exist in the issue tracker.
- [X] I have read and understood the contribution guidelines.

### PR DESCRIPTION
**Pull request by MaartenBW**
## Description

Please write a description of the changes you made here.

Fixes #167 

## Type of change

Please delete options that are not relevant.

-  New feature (non-breaking change which adds functionality)

## Checklist:

- [ x] I have added tests that prove my fix is effective or that my feature works
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_14.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def __init__(
         ----------
         l_0 : M
             [:math:`l_{0}`] Effective length [:math:`m`].
+            Use your own implementation of this value or use :class: `Form5Dot15EffectiveLengthBraced`.
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_14.py:29]: add 'integration' test to check this.

**Review by bro-wi — APPROVED** [review]
lgtm

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_15.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+        NEN-EN 1992-1-1+C2:2011 art.5.8.3.2(1) - Formula (5.15)
+
+        Parameters
+        ----------
+        k_1 : -
+            [:math:`k_{1}`] Relative flexibility of rotational constraint at end 1 [:math:`-`].
+        k_2 : -
+            [:math:`k_{1}`] Relative flexibility of rotational constraint at end 2 [:math:`-`].
+        l : M
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_15.py:29]:
```suggestion
        height : M
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_14.py (hunk 1 above)
Comment (MaartenBW) [reply blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_14.py:29]: Do you have a example of this? I see this sentence a lot, but dont see any integration tests.
And what is the point? In the end, it is just a value plugged into a formula.

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_15.py (hunk 2 above)
Comment (MaartenBW) [reply blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_15.py:29]: Fixed

**Pull request merged**